### PR TITLE
Food stacks

### PR DESCRIPTION
**Pull request by MaxIsJoe**
Food now lasts longer and crew members can no longer eat like snakes.

Before this PR:
https://youtu.be/zAwcj6d8XTQ

After this PR:

https://user-images.githubusercontent.com/34368774/215391986-3e772146-9d20-407d-a896-f76755fac3d5.mp4

### Changelog:

CL: [New] Food now have bite stacks which allows them to last longer.
CL: [Fix] Removed unnecessary debug logs related to hover tooltips.

### CODE REVIEW

File: UnityProject/Assets/Scripts/Items/Food/XenomorphFood.cs [hunk 1]
```diff
@@ -52,7 +52,7 @@ public override void TryConsume(GameObject feederGO, GameObject eaterGO)
 			}
 		}
 
-		public override void Eat(PlayerScript eater, PlayerScript feeder)
+		protected override void Eat(PlayerScript eater, PlayerScript feeder)
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Items/Food/XenomorphFood.cs:55]: Doesn't this mean the contents is duplicated each time you eat?

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Items/Food/XenomorphFood.cs:55]: no

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Items/Food/XenomorphFood.cs:55]: where does it split the contents of the Reagent mix of the stomach?

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Items/Food/XenomorphFood.cs:55]: *of the food

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Items/Food/XenomorphFood.cs:55]: Go to the edible component and see for yourself

File: UnityProject/Assets/Scripts/Items/Food/Edible.cs [hunk 2]
```diff
@@ -176,9 +184,16 @@ public virtual void Eat(PlayerScript eater, PlayerScript feeder)
 			}
 			else
 			{
-				_ = Inventory.ServerDespawn(gameObject);
+				Bite(feeder, feederSlot);
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Items/Food/Edible.cs:187]: don't this still just duplicate the reagent mix?

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Items/Food/Edible.cs:187]: Are you talking about line 169? Please be more specific about what you're talking about

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Items/Food/Edible.cs:187]: "Stomach.StomachContents.Add(incomingFood.Clone());" 
yeah, this is called each time for each eat?

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Items/Food/Edible.cs:187]: ye

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Items/Food/Edible.cs:187]: So the content gets depplycaded?

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Items/Food/Edible.cs:187]: yes

**Review by corp-0 — APPROVED** [review]

**Pull request merged**